docs/src/main/sphinx/release/release-411.md

```diff
@@ -0,0 +1,59 @@
+# Release 411 (15 Mar 2023)
+
+## General
+
+* Add spilled data size to query statistics. ({issue}`16442`)
+* Add {func}`sinh` function. ({issue}`16494`)
+* Add support for a `GRACE PERIOD` clause in the `CREATE MATERIALIZED VIEW`
+  task. This is a backwards incompatible change, and the previous behavior can
+  temporarily be restored with the `legacy.materialized-view-grace-period`
+  configuration property or the `legacy_materialized_view_grace_period` session
+  property. ({issue}`15842`)
+* Add `CANCELING`, `ABORTING`, and `FAILING` task statuses for tasks which are
+  waiting on running drivers to fully terminate. ({issue}`15478`)
+* Improve error messages for exceptions thrown by a connector while querying
+  metadata. ({issue}`16497`)
+* Fix generation of IDs for `keyvalue()` methods in the
+  [JSON path](json-path-language). Previously, separate IDs were generated for
```

Comment:
Can we leave this sentence out .. since we dont actually say what it does now anyway?

Reply:
#16482 looks like a correctness bugfix, so we should keep it, but we need to reword in user-understanable terms. is it "fix incorrect query results when using JSON path `keyvalue` function"?

cc @electrum